Cleanse/Graph.swift

```diff
@@ -99,7 +99,7 @@ class Graph : BinderBase {
 
         if type is AnyWeakProvider.Type {
             return Provider(value: type.makeNew(getter: { [weak futureProvider] in
-                 futureProvider?.getAny()
+                 futureProvider?.getAny() as Any
```

Comment:
I've been a bit wary about this line. I know it is yielding a warning, but not sure what the proper fix is. I don't think we have test coverage on this line iether. Change OK though